layouts/index.html

```diff
@@ -107,7 +108,8 @@ <h3><a class="stretched-link" style="color: #222222;" href="{{ .RelPermalink }}"
       <h2 class="section__blog-title text-center my-0">MetalBear Writers Program</h2>
     </header>
     <div class="text-center">
-      <p>Submit a brief application with a topic of interest and an outline, along with a writing sample that showcases your experience and writing skills?</p>
+      <p>Submit a brief application with a topic of interest and an outline, along with a writing sample that showcases
+        your experience and writing skills?</p>
```

Comment:
not new but let's fix
```suggestion
        your experience and writing skills</p>
```